src/commands/Admin/roleset.ts

```diff
@@ -16,15 +16,29 @@ import { ApplyOptions } from '@skyra/decorators';
 	usage: '<add|remove|reset|list|auto:default> (name:name) (role:rolenames)',
 	usageDelim: ' '
 })
+@CreateResolvers([
+	[
+		'name', (arg, possible, message, [subcommand]) => {
+			if (subcommand === 'list') return undefined;
+			if (!arg && subcommand === 'reset') return undefined;
+			return message.client.arguments.get('string')!.run(arg, possible, message);
+		}
+	],
+	[
+		'rolenames', (arg, possible, message, [subcommand]) => {
+			if (subcommand === 'list' || subcommand === 'reset') return undefined;
+			return message.client.arguments.get('rolenames')!.run(arg, possible, message);
+		}
+	]
+])
```

Comment:
Is this really the ESLint suggested style? Seems kinda bulky with the [ ] having new lines.

Reply:
ESLint doesn't nag about it anyway. I like this style because it's readable.